llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1341,7 +1341,102 @@ void SpirvLowerGlobal::addCallInstForOutputExport(Value *outputValue, Constant *
     m_builder->CreateWriteGenericOutput(outputValue, location, locOffset, elemIdx, maxLocOffset, outputInfo, vertexIdx);
   }
 }
+Value *SpirvLowerGlobal::getConstantInt32(unsigned c) {
+  return ConstantInt::get(Type::getInt32Ty(*m_context), c);
+}
+// =====================================================================================================================
+// Inserts instructions to load needed members from input/ouput location.
```

Comment:
Since this function is very similar to loadInOutMember, we ought to make some comments to explain their difference in uses. Also, if you assume it is only used to get interpolant members, you can make the function name more specific, such as loadInterpolantInputMembers. If you believe this function should stay more generally, it is OK. We just describe its particularity by comparing with loadInoutMember

Reply:
Yes, the comment is meaningful. But loadInterpolantInputMembers is too specific and hard to image what's going on internally. Maybe a better function name is needed. I will add some comment.

Reply:
I just changed the function name to loadDynamicIndexedMembers(), and also add some comment to explain how the function works. please check whether you are happy with the new version.